### PR TITLE
Note about how remove unused images (bsc#1198654)

### PR DESCRIPTION
**Pull request by s0nea**
Signed-off-by: Tatjana Dehler <tdehler@suse.com>

### CODE REVIEW

**Comment by s0nea** [comment]
I don't know why the bot keeps adding its commit. Even though I added the change to my commit, already.

File: xml/upgrade-to-pacific.xml [hunk 1]
```diff
@@ -192,6 +192,21 @@
...
     </para>
+    <tip>
+     <title>Remove unused container images</title>
+     <para>
+      To remove unused container images remaining on the system after the
+      upgrade, run:
+     </para>
+<screen>
+&prompt.root;salt '*' cmd.shell "podman image prune --all --force"
+</screen>
```
Comment (tbazant) [inline xml/upgrade-to-pacific.xml:203]:
```suggestion
<screen>&prompt.smaster;salt '*' cmd.shell "podman image prune --all --force"</screen>
```

File: xml/upgrade-to-pacific.xml [hunk 2]
```diff
@@ -192,6 +192,21 @@
      linkend="deploy-cephadm-configure-registry"/> for more details
      about configuring a local container image registry.
     </para>
+    <tip>
+     <title>Remove unused container images</title>
+     <para>
+      To remove unused container images remaining on the system after the
+      upgrade, run:
```
Comment (tbazant) [inline xml/upgrade-to-pacific.xml:199]:
```suggestion
      Optionally, remove unused container images remaining on the system after the
      upgrade:
```

File: xml/upgrade-to-pacific.xml [hunk 3]
```diff
@@ -192,6 +192,21 @@
...
+     </para>
+<screen>
+&prompt.root;salt '*' cmd.shell "podman image prune --all --force"
+</screen>
+     <para>
+      Please note: the step is <emphasis role="bold">optional</emphasis> and
+      <emphasis role="bold">not</emphasis> mandatory. You can of course keep
+      the unused images on your system if you prefer.
+     </para>
```
Comment (tbazant) [inline xml/upgrade-to-pacific.xml:208]:
```suggestion
```

**Review by tbazant — CHANGES_REQUESTED** [review]
Consider my changes to streamline the flow

**Comment by s0nea** [comment]
> Consider my changes to streamline the flow

Thank you for the review! I'll update the PR accordingly.

File: xml/upgrade-to-pacific.xml [hunk 4]
```diff
@@ -192,6 +192,14 @@
      linkend="deploy-cephadm-configure-registry"/> for more details
      about configuring a local container image registry.
     </para>
+    <tip>
+     <title>Remove unused container images</title>
+     <para>
+      Optionally, remove unused container images remaining on the system after
+      the upgrade:
+     </para>
+<screen>&prompt.root;salt '*' cmd.shell "podman image prune --all --force"</screen>
```
Comment (tbazant) [inline xml/upgrade-to-pacific.xml:201]:
```suggestion
<screen>&prompt.smaster;salt '*' cmd.shell "podman image prune --all --force"</screen>
```

**Review by tbazant — APPROVED** [review]
LGTM

**Comment by tbazant** [comment]
> I don't know why the bot keeps adding its commit. Even though I added the change to my commit, already.

It's probably XML formatting webhook, just ignore it

**Review by dariavladykina — APPROVED** [review]
LGTM

**Pull request merged**